src/core/reactor.cc

```diff
@@ -206,17 +206,26 @@ shard_id reactor::cpu_id() const {
 
 io_priority_class
 reactor::register_one_priority_class(sstring name, uint32_t shares) {
```

Comment:
Shouldn't these functions be marked deprecated as well? Maybe it is, in a different patch?

Reply:
They were (well -- should have been) in a different patch, yes. However now I'm reworking the PR and this function if ifdef-ed out